tensorflow_probability/python/distributions/loglogistic_test.py

```diff
@@ -0,0 +1,209 @@
+      self.evaluate(dist.mean()),
+      stats.fisk.mean(loc=0., scale=log_logistic_scale, c=1./scale)
+    )
+
+  def testLogLogisticMeanNoNanAllowed(self):
+    log_logistic_scale = np.float32([3., 1.5, 0.75])
+    loc = np.log(log_logistic_scale)
+    scale = np.float32([0.4, 0.6, 1.5])
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True,
+                           allow_nan_stats=False)
+
+    with self.assertRaisesOpError('Condition x < y.*'):
+      self.evaluate(dist.mean())
+
+  def testLogLogisticVariance(self):
+    log_logistic_scale = np.float32([3., 1.5, 0.75])
+    loc = np.log(log_logistic_scale)
+    scale = np.float32([0.4, 0.3, 1.5])
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True)
+
+    self.assertAllClose(
+      self.evaluate(dist.variance()),
+      stats.fisk.var(loc=0., scale=log_logistic_scale, c=1./scale)
+    )
+    self.assertAllClose(
+      self.evaluate(dist.stddev()),
+      stats.fisk.std(loc=0., scale=log_logistic_scale, c=1./scale)
+    )
+
+  def testLogLogisticVarianceNoNanAllowed(self):
+    log_logistic_scale = np.float32([3., 1.5, 0.75])
+    loc = np.log(log_logistic_scale)
+    scale = np.float32([0.4, 0.6, 1.5])
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True,
+                           allow_nan_stats=False)
+
+    with self.assertRaisesOpError('Condition x < y.*'):
+      self.evaluate(dist.variance())
+
+    with self.assertRaisesOpError('Condition x < y.*'):
+      self.evaluate(dist.stddev())
+
+  def testLogLogisticMode(self):
+    log_logistic_scale = np.float32([3., 1.5, 0.75])
+    loc = np.log(log_logistic_scale)
+    scale = np.float32([0.4, 0.6, 1.5])
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True)
+
+    mode = log_logistic_scale * ((1. - scale) / (1. + scale)) ** scale
+    mode[2] = 0.
+    self.assertAllClose(
+      self.evaluate(dist.mode()),
+      mode
+    )
+
+  def testLogLogisticEntropy(self):
+    log_logistic_scale = np.float32([3., 1.5, 0.75])
+    loc = np.log(log_logistic_scale)
+    scale = np.float32([0.4, 0.6, 1.5])
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True)
+
+    self.assertAllClose(
+      self.evaluate(dist.entropy()),
+      stats.fisk.entropy(loc=0., scale=log_logistic_scale, c=1./scale)
+    )
+
+  def testLogLogisticSample(self):
+    log_logistic_scale = 1.5
+    loc = np.log(log_logistic_scale).astype(np.float32)
+    scale = 1./3
+    dist = tfd.LogLogistic(loc=loc, scale=scale, validate_args=True)
+    samples = self.evaluate(dist.sample(6000, seed=test_util.test_seed()))
+    self.assertAllClose(np.mean(samples),
+                        self.evaluate(dist.mean()),
+                        atol=0.1)
+    self.assertAllClose(np.std(samples),
+                        self.evaluate(dist.stddev()),
+                        atol=0.5)
+
+  def testLogLogisticPDF(self):
+    log_logistic_scale = 1.5
+    loc = np.log(log_logistic_scale)
+    scale = 1./0.4
```

Comment:
Use batches and broadcasting for these pdf/cdf related tests? Could also add separate tests where you have the parameters loc and scale broadcast, so you have a non-trivial batch of LogLogistic distributions.